konfera/templates/konfera/base.html

```diff
@@ -26,7 +26,8 @@
 
   {% if messages %}
     {% for message in messages %}
-    <div class="alert alert-success alert-dismissible{% if message.tags %} {{ message.tags }}{% endif %}" role="alert">
+    <div class="alert alert-{{ message.tags }} alert-dismissible{% if message.tags %} {{ message.tags }}{% endif %}"
```

Comment:
We don't need `{% if message.tags %} {{ message.tags }}{% endif %}` do we? 
I mean when a tag is set it shows up as `alert-XYZ` which is fine for our design.